lib/web/fetch/util.js

```diff
@@ -1055,6 +1055,11 @@ async function fullyReadBody (body, processBody, processBodyError) {
   //    with taskDestination.
   const errorSteps = processBodyError
 
+  if (typeof body.source === 'string') {
+    successSteps(new TextEncoder().encode(body.source))
```

Comment:
In this way, test failures can be corrected.
```suggestion
    const buffer = new TextEncoder().encode(body.source)
    successSteps(Buffer.from(buffer.buffer, buffer.byteOffset, buffer.byteLength))
```

Reply:
thank you for suggestion 🙏

Reply:
@tsctx I wonder how you caught this

Reply:
Because I had the same error.